examples/select/select.go

```diff
@@ -9,30 +10,32 @@ import "fmt"
 
 func main() {
 
-    // For our example we'll select across two channels.
-    c1 := make(chan string)
-    c2 := make(chan string)
+	// Per il nostro esempio utilizzeremo select su due
+	// canali.
+	c1 := make(chan string)
+	c2 := make(chan string)
 
-    // Each channel will receive a value after some amount
-    // of time, to simulate e.g. blocking RPC operations
-    // executing in concurrent goroutines.
-    go func() {
-        time.Sleep(time.Second * 1)
-        c1 <- "one"
-    }()
-    go func() {
-        time.Sleep(time.Second * 2)
-        c2 <- "two"
-    }()
+	// Ogni canale riceverà un valore dopo un po' di
+	// tempo, giusto per simulare ad esempio operazioni
+	// RPC bloccanti eseguite su multiple goroutine.
+	go func() {
+		time.Sleep(time.Second * 1)
+		c1 <- "uno"
+	}()
+	go func() {
+		time.Sleep(time.Second * 2)
+		c2 <- "due"
+	}()
 
-    // We'll use `select` to await both of these values
-    // simultaneously, printing each one as it arrives.
-    for i := 0; i < 2; i++ {
-        select {
-        case msg1 := <-c1:
-            fmt.Println("received", msg1)
-        case msg2 := <-c2:
-            fmt.Println("received", msg2)
-        }
-    }
+	// Utilizzeremo `select` per ricevere dai due canali
+	// simultaneamente, stampando ogni valore ogni qual
```

Comment:
Si scrive _ogniqualvolta_. Cmq io cambierei in `stampando ogni valore che arriva`.